### DISCUSSION TITLE
How to update the audio length when the src changes?

### DISCUSSION THREAD

**Discussion by machadoug** [yoriiis/vlitejs#93] [Q&A] [answered]
I'm using the same <audio> for several audio files.  However, when we change the audio file the audio length in the player stays the same.
How can I update the Player's audio length when the src changes? I believe this should be included in the core.

**Comment by yoriiis** [comment, marked as answer]
The easiest way is to destroy the instance, inject the new player (with the new source) and rebuild the player instance.
